lib/interceptor/redirect.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const RedirectHandler = require('../handler/redirect')
+
+function RedirectInterceptor ({ maxRedirections: defaultMaxRedirections }) {
```

Comment:
this has class naming but is not a class. maybe it should be renamed to indicate that this is a factory function for a partially resolved function?